### PR TITLE
Add 1.1 release notes

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Added release notes for the 1.1 Helsinki release

**Related issue(s)**
See also #272

### CODE REVIEW

File: content/blog-posts/2019-05-13-release-notes-1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### Event Service
+
+We enriched the Event Service with the new `/{application}/v1/events/subscribed` endpoint that only returns information on the subscribed Events. This endpoint is perfect for your system optimization as the connected application no longer needs to send Events that are not used by any lambda or service. [Read more](/docs/1.1/components/application-connector/#api-event-service) about the endpoint and learn how you can use it to fetch Events.
+
+## Console
+
+### Performance improvements  
+
+As a result of aggregating several calls into one, we reduced the initial loading time of the Console UI. We also leveraged a new Luigi feature for improving caching behavior when reopening micro front-ends. Thanks to it, micro front-ends no longer need to reload each time you switch from one to another.
```
Comment (derberg) [inline content/blog-posts/2019-05-13-release-notes-1.1/index.md]: `As a result of aggregating several calls into one, we reduced the initial loading time of the Console UI` which views?

File: content/blog-posts/2019-05-13-release-notes-1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+![Namespace overview](./namespaces-overview.png)
+
+## Core and Supporting
+
+### Minio Gateway mode
+
+The [Asset Store](/docs/1.1/components/asset-store) that provides asset management in Kyma uses [Minio](https://min.io/) as a back-end solution. As stated in our documentation, we recommend that you use Minio in its Gateway mode for your production environment. This means you should use Minio as a gateway to Google Cloud Storage (GCS) and other file storage providers.
+
+In this release we focused on preparing an easy switch from the standalone mode to the Gateway mode by:
+- Making sure sure your data is seamlessly recreated after the switch
```
Comment (derberg) [inline content/blog-posts/2019-05-13-release-notes-1.1/index.md]: please add another bullet that we also integrated Minio Gateway mode with our testing pipeline, so we can now always test any new functionality against Minio Gateway mode pointing to Google Cloud Storage

File: content/blog-posts/2019-05-13-release-notes-1.1/index.md (hunk 1 above)
Comment (kazydek) [reply content/blog-posts/2019-05-13-release-notes-1.1/index.md]: Console UI navigation

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+---
+
+After the long-awaited 1.0 Gliwice release, we focused mainly on putting the polishing touches to the current setup, including improvements in the Console UI performance and usability, Asset Store and Service Catalog extensions, Application Operator optimization, and documentation-related tweaks. Still, we are proud to communicate a few new features we bring to you in 1.1. Helsinki. Read on to find out what has changed in Kyma since 1.0.
+
+<!-- overview -->
+
+The highlights of Kyma 1.1 Helsinki include:
+
+- [Add-on Catalog](#service-management) - We enabled a new way to share add-ons by combining the concepts of Service Catalog bundles with Helm charts.
+- [AWS Broker](#asset-store) - We extended our third-party services by adding the AWS service offering.
```
Comment (derberg) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]: I think link is wrong, for the others too

Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: yup, copied and forgot to change it, thanks for noticing it ;)

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+- [Productivity](#productivity) - Octopus in Kyma
+- [Service Management](#service-management) - Add-ons in the Catalog UI, AWS Service Broker add-on, documentation support for add-ons, "provision only once" APIs and Events
+
+---
+
+## Application Connector
+
+### Application Operator optimization
+
+In this release, we optimized the memory consumption for the Application Operator. The component is now much more stable and reliable even under the high cluster load.
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
In this release, we optimized the memory consumption for the Application Operator. The component is now much more stable and reliable, even under the high cluster load.
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+## Console
+
+### Performance improvements  
+
+As a result of aggregating several calls into one, we reduced the initial loading time of the Console UI navigation. We also leveraged a new Luigi feature for improving caching behavior when reopening micro front-ends. Thanks to it, micro front-ends no longer need to reload each time you switch from one to another.
+
+### Usability improvements
+
+We added the `SYSTEM` badge to the system Namespaces thanks to which you can easily distinguish them from the user-created ones on the Namespace overview page in the Console UI.
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
We added the `SYSTEM` badge to the system Namespaces, thanks to which you can easily distinguish them from the user-created ones on the Namespace overview page in the Console UI.
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+### Minio Gateway mode
+
+The [Asset Store](/docs/1.1/components/asset-store) that ensures asset management in Kyma uses [Minio](https://min.io/) as a back-end solution. As stated in our documentation, we recommend that you use Minio in its Gateway mode for your production environment. This means you should use Minio as a gateway to Google Cloud Storage (GCS).
+
+In this release, we focused on preparing an easy switch from the standalone mode to the Gateway mode by:
+- Making sure your data is seamlessly recreated after the switch
+- Improving the stability of the Asset Store with Minio in the Gateway mode
+- Providing clear documentation on how to [switch to GCS](/docs/1.1/components/asset-store/#tutorials-set-minio-to-the-google-cloud-storage-gateway-mode)
+- Integrating the Minio Gateway mode with our testing pipeline. This way, we can now test any new functionality against the Minio Gateway mode pointing to GCS
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
- Integrating the Minio Gateway mode with our testing pipeline. This way, we can now test any new functionality against the Minio Gateway mode pointing to GCS.
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+
+## Documentation
+
+### Configuration and installation document improvements
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
### Configuration and installation documents improvements
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+In 1.1, we focused on creating configuration documents and simplifying the existing installation documents.
+
+Configuration improvements:
+- An [overview](/docs/1.1/root/kyma/#configuration-overview) document explaining what you can configure in Kyma and how you can do it before and after installation.
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
- An [overview](/docs/1.1/root/kyma/#configuration-overview) document explaining what you can configure in Kyma and how you can do it before and after installation
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+- Improved documents on selected [component installation](/docs/1.1/root/kyma/#configuration-custom-component-installation) and chart values [overrides](/docs/1.1/root/kyma/#configuration-helm-overrides-for-kyma-installation).
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
- Improved documents on selected [component installation](/docs/1.1/root/kyma/#configuration-custom-component-installation) and chart values [overrides](/docs/1.1/root/kyma/#configuration-helm-overrides-for-kyma-installation)
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+- A set of configuration documents for the [Asset Store](/docs/1.1/components/asset-store/#configuration-configuration) specifying the configurable parameters from the `values.yaml` charts and sub-charts that you can override. You can expect more of such documents for other Kyma components to come in the upcoming releases.
+
+Installation improvements:
+- Simplified [cluster installation](/docs/1.1/root/kyma/#installation-install-kyma-on-a-cluster) documents explaining how you can can quickly deploy Kyma on a cluster with a wildcard DNS provided by `xip.io`.
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
- Simplified [cluster installation](/docs/1.1/root/kyma/#installation-install-kyma-on-a-cluster) documents explaining how you can quickly deploy Kyma on a cluster with a wildcard DNS provided by `xip.io`
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+![Add-ons 2](./add-ons-2.png)
+
+### AWS Service Broker add-on
+
+As part of our constant efforts to improve your experience on third-party Service Brokers in Kyma, we extended our Service Catalog with Amazon offerings. The new [AWS Service Broker](/docs/1.1/components/service-catalog/#service-brokers-aws-service-broker) is available for you as an add-on, extending the existing GCP and Azure Service Brokers add-on family.
+
+### Documentation support for add-ons
+
+The Helm Broker creates Service Class documentation from an add-on which appears in the **Add-Ons** Catalog UI view. Read how to add documentation to your [add-on](https://kyma-project.io/docs/master/components/helm-broker/#details-create-a-bundle-docs-directory). If your add-on extends the Service Catalog UI, read how to provide documentation for those new [Service Classes](/docs/1.1/components/helm-broker#details-service-classes-documentation-provided-by-bundles).
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
The Helm Broker creates Service Class documentation from an add-on which appears in the **Add-Ons** Catalog UI view. [Read](https://kyma-project.io/docs/master/components/helm-broker/#details-create-a-bundle-docs-directory) how to add documentation to your add-on. If your add-on extends the Service Catalog UI, [read]((/docs/1.1/components/helm-broker#details-service-classes-documentation-provided-by-bundles) how to provide documentation for those new Service Classes.
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+![Provision only once](./provision-only-once.png)
+
+
+## Productivity
+
+### Octopus in Kyma
+
+As part of productivity improvements in Kyma, we replaced Helm tests with the Octopus test runner. Kyma components now use Octopus as a testing framework to run tests defined as Docker images on a running cluster.
+Read more about [testing ](/docs/1.1/root/kyma/#details-testing-kyma) in Kyma, [Octopus](https://github.com/kyma-incubator/octopus/blob/master/README.md) itself, and the benefits it brings to the Kyma testing process.
```
Comment (klaudiagrz) [inline content/blog-posts/2019-05-15-release-notes-1.1/index.md]:
```suggestion
Read more about [testing](/docs/1.1/root/kyma/#details-testing-kyma) in Kyma, [Octopus](https://github.com/kyma-incubator/octopus/blob/master/README.md) itself, and the benefits it brings to the Kyma testing process.
```

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: Yeah, I corrected that already ;/

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+  - "/blog/release-notes-11"
+---
+
+After the long-awaited 1.0 Gliwice release, we focused mainly on putting the polishing touches to the current setup, including improvements in the Console UI performance and usability, Asset Store and Service Catalog extensions, Application Operator optimization, and documentation-related tweaks. Still, we are proud to communicate a few new features we bring to you in 1.1. Helsinki. Read on to find out what has changed in Kyma since 1.0.
+
+<!-- overview -->
+
+The highlights of Kyma 1.1 Helsinki include:
+
+- [Add-on Catalog](#service-management) - We enabled a new way to share add-ons by combining the concepts of Service Catalog bundles with Helm charts.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: yup, same here

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+- [AWS Broker](#asset-store) - We extended our third-party services by adding the AWS service offering.
+- [Minio Gateway](#asset-store) - We improved the Asset Store configuration to let you use Google Cloud Storage.
+- [Octopus](#productivity) - We introduced a new testing tool that replaced Helm tests.
+
+See the overview of all changes in this release:
+
+- [Application Connector](#application-connector) - Application Operator optimization, OData support in documentation, new Event Service endpoint
+- [Console](#console) - Performance improvements, usability improvements
+- [Core and Supporting](#core-and-supporting) - Minio Gateway mode
+- [Documentation](#documentation) - Configuration and installation document improvements
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: I would leave it as it is as I don't think that is a mistake ...?

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md (hunk 5 above)
Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: I started a new sentence with it :)

File: content/blog-posts/2019-05-15-release-notes-1.1/index.md (hunk 7 above)
Comment (kazydek) [reply content/blog-posts/2019-05-15-release-notes-1.1/index.md]: as above

**Pull request merged**